src/error-helpers.ts

```diff
@@ -25,3 +25,7 @@ export function formatErrorCtor(
     error
   );
 }
+
+export function errorMatch(lines: RegExp[]): RegExp {
```

Comment:
Im wondering if this should be in a utils folder under `__tests__` since its used only for tests